File: app/src/main/java/com/snakes_ladder/dice/PreLollipopSoundPool.java
package com.snakes_ladder.dice;

import android.media.AudioManager;
import android.media.SoundPool;

/**
 * Created a pre Lollipop SoundPool
 */
final class PreLollipopSoundPool {
    @SuppressWarnings("deprecation")
    public static SoundPool NewSoundPool() {
        return new SoundPool(1, AudioManager.STREAM_MUSIC,0);
    }
}

File: app/src/main/java/com/snakes_ladder/dice/PreLollipopSoundPool.kt
package com.snakes_ladder.dice

import android.media.AudioManager
import android.media.SoundPool

/**
 * Created a pre Lollipop SoundPool
 */
internal object PreLollipopSoundPool {
    fun NewSoundPool(): SoundPool {
        return SoundPool(1, AudioManager.STREAM_MUSIC, 0)
    }
}